.editorconfig

```diff
@@ -0,0 +1,17 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[package.json]
+indent_style = space
+indent_size = 2
```

Comment:
If we're going to add this, it should be in a separate PR.

Reply:
sorry about that, I didn't mean to add that. I was trying to follow the current style and that file made my editor honor it.